frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -73,14 +71,6 @@ export const VMSettingsTabComponent: React.FC<VMSettingsTabComponentProps> = ({
           />
         </FormField>
       </FormFieldMemoRow>
```

Comment:
how will the user know which template was selected in `virtualization/templates` ?

Reply:
what do you propose ? keep the template's dropdown or add a label (or another component) to show the chosen template ?

Reply:
I guess having a label should be enough. Although there will probably be some logic , when the template doesn't exist, etc. So having a component might be beneficial.